src/pages/Authentication/Login.js

```diff
@@ -0,0 +1,130 @@
+import "./Authentication.css";
+import { useState } from "react";
+import { Link, useLocation, useNavigate } from "react-router-dom";
+import { Box } from "../../components";
+import { userLogin } from "../../utils/authenticationCalls";
+import { useAuth } from "../../contexts";
+import { FaEye, FaEyeSlash } from "react-icons/fa";
+
+export const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [showPassword, setShowPassword] = useState(false);
+  const { dispatchAuth } = useAuth();
+  //   const { dispatchData } = useData();
+  const navigate = useNavigate();
+  const location = useLocation();
+  const gotoPath = location.state?.from?.pathname || "/";
+
+  const testData = {
+    email: "adarshbalika@gmail.com",
+    password: "adarshBalika123",
+  };
+
```

Comment:
You can change the default user to your own name and email

Reply:
Thanks for this suggestion Junaid! made the changes